src/chainparams.cpp

```diff
@@ -375,7 +383,12 @@ class CRegTestParams : public CChainParams {
         base58Prefixes[SECRET_KEY] =     std::vector<unsigned char>(1,239);
         base58Prefixes[EXT_PUBLIC_KEY] = {0x04, 0x35, 0x87, 0xCF};
         base58Prefixes[EXT_SECRET_KEY] = {0x04, 0x35, 0x83, 0x94};
+    
+        vPremineAddressWhitelist = {
+            "2N7eDqMJfmtUPFEZumx81HqN4YKXyD4CmLU",
```

Comment:
And this.

Reply:
I will change it in the address format PR.